hypervisor/arch/x86/guest/vcpuid.c

```diff
@@ -29,7 +29,7 @@ static inline struct vcpuid_entry *local_find_vcpuid_entry(const struct acrn_vcp
 				(tmp->subleaf != subleaf)) {
 				continue;
 			}
-			entry = tmp;
+			entry = (const struct vcpuid_entry *)tmp;
```

Comment:
should we change the tmp to be const * too?  The contents it points to is not changed.

Reply:
OK. Then we need to convert &vm->vcpuid_entries[i] to const